moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -47,17 +48,22 @@ include_directories(
 ## Library to process realtime twist or joint commands ##
 #########################################################
 
+# This library provides a way of loading parameters for servo
+add_library(${SERVO_PARAM_LIB_NAME} SHARED src/servo_parameters.cpp)
+set_target_properties(${SERVO_PARAM_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+ament_target_dependencies(${SERVO_PARAM_LIB_NAME} ${THIS_PACKAGE_INCLUDE_DEPENDS})
+
 # This library provides an interface for sending realtime twist or joint commands to a robot
 add_library(${SERVO_LIB_NAME} SHARED
   # These files are used to produce differential motion
   src/servo.cpp
   src/servo_calcs.cpp
   src/collision_check.cpp
   src/low_pass_filter.cpp
-  include/moveit_servo/servo_parameters.cpp
 )
 set_target_properties(${SERVO_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 ament_target_dependencies(${SERVO_LIB_NAME} ${THIS_PACKAGE_INCLUDE_DEPENDS})
+target_link_libraries(${SERVO_LIB_NAME} ${SERVO_PARAM_LIB_NAME})
 
 # Add and export library for composable node server
```

Comment:
`# Add and export library to run as a ROS node component, and receive commands via topics`